client/verta/verta/registry/entities/_modelversion.py

```diff
@@ -499,23 +585,25 @@ def log_environment(self, env, overwrite=False):
 
         """
         if not isinstance(env, _Environment):
-            raise TypeError("`env` must be of type Environment, not {}".format(type(env)))
+            raise TypeError(
+                "`env` must be of type Environment, not {}".format(type(env))
+            )
 
         if self.has_environment and not overwrite:
-            raise ValueError("environment already exists; consider setting overwrite=True")
+            raise ValueError(
+                "environment already exists; consider setting overwrite=True"
+            )
 
         if overwrite:
             self._fetch_with_no_cache()
             self._msg.environment.CopyFrom(env._as_env_proto())
             self._update(self._msg, method="PUT")
         else:
-            self._update(self.ModelVersionMessage(environment=env._as_env_proto()), method="PATCH",
```

Comment:
Apologies for the autoformat complications, but this is the one significant change here

Reply:
That certainly is simplify simpler than what it was before!